contracts/staking/validatorShare/ValidatorShare.sol

```diff
@@ -147,6 +147,10 @@ contract ValidatorShare is IValidatorShare {
             _amount = _amount.sub(_rewards);
         }
 
+        if (_amount > activeAmount) {
+            _amount = activeAmount;
```

Comment:
This is wrong.

Reply:
Can you elaborate what is wrong and how would you do this?

Reply:
bob stakes 200 and alice stakes 100, after which total pool 300 got slashed by 50% then we don't want to give bob 150 instead it should be 100(bob) and 50(Alice).